### ISSUE TITLE
README

### ISSUE THREAD

**Issue by pym96** [xyimaging/FNPC#3]
Hi, can you give this repo a README file? I'll appreciate it if you can help me.

**Comment by xyimaging** [comment]
Certainly! I will promptly update the README file. Thank you for your interest in our project!
